src/Views/batchprocessor.html

```diff
@@ -277,20 +277,28 @@ <h3>Select Basin Characteristics</h3>
 <div class="modal-footer">
     <div class="row" style="justify-content: center;">
         <!-- Button to submit batch -->
-        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch'" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
+        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch' && vm.submitBatchOver250 == false" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
             ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i
 
                 class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch</span></button>
         <!-- End submit batch button -->
 
+        <!-- Button to submit GTE 250 batch -->
+        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch' && vm.submitBatchOver250 == true" type="submit"
+            class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch(true)"
+            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i
+                class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch Over 250 Points</span></button>

```

Comment:
Turn the fa-share icon into a spinning loader icon while the batch is being submitted, since large batches take a minute to process.